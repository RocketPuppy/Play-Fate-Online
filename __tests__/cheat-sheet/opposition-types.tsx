import * as React from "react";
import * as renderer from "react-test-renderer";
import OppositionTypes from "../../src/cheat-sheet/opposition-types";

test("renders all the pieces", () => {
  const render = renderer.create(<OppositionTypes />);

  expect(render.toJSON()).toMatchSnapshot();
});
