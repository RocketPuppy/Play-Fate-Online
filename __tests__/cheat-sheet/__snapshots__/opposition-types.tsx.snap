// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Opposition Types
  </h2>
  <dl>
    <dt>
      Active
    </dt>
    <dd>
      another character rolls against you
    </dd>
    <dt>
      Passive
    </dt>
    <dd>
      a static rating on the ladder
    </dd>
  </dl>
</div>
`;
